dotCMS/src/integration-test/java/com/dotcms/datagen/LanguageCodeDataGen.java

```diff
@@ -0,0 +1,32 @@
+package com.dotcms.datagen;
+
+import java.util.Locale;
+import java.util.Random;
+
+/**
+ * Generater a language code
+ * @author jsanca
+ */
+public class LanguageCodeDataGen implements DataGen<String> {
+
+    final String[] languageCodes = Locale.getISOLanguages();
+    final Random   random        = new Random();
+
+    @Override
+    public String next() {
+
+        final int    languageIndex = this.random.nextInt(languageCodes.length);
+        final String languageCode  = this.languageCodes[languageIndex];
+        return languageCode;
+    }
+
+    @Override
+    public String persist(String object) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Parameter 'object' is not assigned and could be declared final](https://app.codacy.com/app/dotCMS/core/pullRequest?prid=4632905)